operator/deploy/operator-psp.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kiali-operator-psp
+  labels:
+    app: kiali-operator
+    version: v1.1.0
+spec:
```

Comment:
@mwringe can you review this PSP spec. I'm not familiar with the security implications here - perhaps you do?